docs/getting-started/bot.md

```diff
@@ -0,0 +1,412 @@
+form `MAJOR.MINOR.PATCH`, e.g., "1.2.3".  Within each Rug archive
+grouping is a list of the message the Atomist Bot recognizes to
+trigger the command handler and a brief description of what the
+command handler does.
+
+[semver]: http://semver.org/
+
+To teach the Atomist Bot a new set of skills, we use the `add skills`
+command.  The `add skills` command takes a single argument: the full
+name of the Rug archive containing the set of skills you want the
+Atomist Bot to start responding to.  The full name of a Rug archive is
+the group name, artifact name, and optional version, each joined by a
+colon (:).  Here we will add the skills from the
+`atomist-rugs:github-handlers` Rug archive.  Since we have omitted the
+version from the full name, the Atomist Bot will add the skills from
+the latest version of the Rug archive available.
+
+```
+@atomist add skills atomist-rugs:github-handlers
+```
+
+The Atomist Bot knows where to find all the command handlers available
+to your team, so it will determine the latest version of
+`atomist-rugs:github-handlers`, download it, and recognize the
+commands that Rug archive makes available.  You can see the new skills
+by sending the Atomist Bot the `show skills` message again.
+
+<div class="ss-container">
+  <img src="../images/show-new-skills.png" alt="Atomist Bot Show Skills" class="ss-medium">
+</div>
+
+Your Atomist Bot will now be able to create new issues for you in
+GitHub.
+
+-->
+
+We can create GitHub Issues without leaving Slack by sending a `create
+issue` message to the Atomist Bot.  When running commands like this,
+the Atomist Bot is aware of the context in which it is asked to run
+the skill.  This allows the Atomist Bot to collect less information in
+certain situations.  As an example, let's run the `create issue`
+command in the `#general` channel of your Slack team.
+
+```
+@atomist create issue
+```
+
+The Atomist Bot responds in a thread, collecting the information it
+needs to send to the command handler.
+
+<div class="ss-container">
+  <img src="../images/create-issue-flow-general.png" alt="Creating a GitHub Issue in #general" class="ss-small">
+</div>
+
+Since the `#general` channel is not linked to a specific GitHub
+repository, the Atomist Bot must ask you in what repository you
+want to create the issue.
+
+Compare this to sending the same `create issue` message sent to the
+Atomist Bot in the `#sprocket` channel it created earlier when it
+created the sprocket project for us.  When the Atomist Bot created the
+project and channel, it linked them together.  This linking becomes
+part of the context when the Atomist Bot responds to messages in that
+channel.  In this case, this means that when you run commands that
+require a GitHub repository as input in the `#sprocket` channel, the
+Atomist Bot does not need to ask you for the repository name, it
+already knows and supplies that information to the command handler
+automatically.
```

Comment:
If I weren't aware of what this was doing, I think this whole paragraph would be difficult for me to make sense. It's almost one of those case where a diagram or GIF would be better.